public/js/actions/sources.js

```diff
@@ -35,6 +37,11 @@ function queueSourcesDispatch(dispatch) {
  */
 function newSource(source) {
   return ({ dispatch }) => {
+    if (isEnabled("features.sourceMaps") && source.sourceMapUrl) {
+      networkRequest(source.sourceMapUrl).then(res => {
+      });
+    }
```

Comment:
This seems like the right place to do this. I'm curious where you think we should store sourcemaps though. Technically, they are an important part of the UI, so they should be in redux, but that might make it hard to serialize...